### ISSUE TITLE
[QUESTION]Questions about the source and sink of sulfate

### ISSUE THREAD

**Issue by zhiweistu** [geoschem/geos-chem#1410]
### Ask a question about GEOS-Chem:
Dear GEOSChem developer and GCAP2.0 developer,
I used GCAP 2.0 to run 2014 year sulfate simulation(20140101-20150101). 
I found sulfur in my result is not balance. 

All variables I used and method:
1.variable “EmisSO4_Total” in “HEMCO_diagnostics” file, estimated total year emission.
2.variable “Prod_SO4” in “GEOSChem.ProdLoss” file, estimated total year sulfate production.
3.variable “DryDep_SO4” in “GEOSChem.DryDep” file, estimated total year dry deposition.
4.variable “WetLossConv_SO4” and “WetLossLS_SO4” in “GEOSChem.Wetloss” file, estimated total year wet deposition.
5.variable “SpeciesRst_SO4” in “GEOSChem.Restart” file, estimate changes of sulfate burden in atmosphere
   (Instantaneous sulfate burden in 20150101 subtract 20140101)

Estimated results: (unit: Tg S)
Sulfate Emission:              1.15 
Sulfate Production:          48.75
Sulfate Wetdep:               49.47
Sulfate Drydep:                 3.74
Sulfate burden change:    0.18

you can see that the production of sulfate is smaller than total sulfate deposition, and sulfur is not balance. the source is smaller 3.49Tg S than sink. So I want to ask if I missed some variables, please give me some guidance, thanks.

**Comment by yantosca** [comment]
Thanks for writing @zhiweistu.  I am also tagging @ltmurray who knows more about the GCAP simulation.

Have you accounted for chemical production of SO4 from e.g. SO2?  That might be something to look into.

**Comment by zhiweistu** [comment]
Thanks for reply @yantosca. Description about Variable “Prod_SO4” I used is “Chemical production of SO4”, I think it is the total sulfate chemical production (included from SO2...).
![image](https://user-images.githubusercontent.com/109276335/194114582-ad87bb8a-cea6-4612-8cb2-ac66fd839e2e.png)
Besides, I found values of other SO4 variable chemical production is zero, it also found in merra2-driven GC.
![image](https://user-images.githubusercontent.com/109276335/194116229-6805617a-1ab7-4ea3-af44-17205116f054.png)
(more detailed description is in https://github.com/geoschem/geos-chem/issues/1401 )
Thanks for your helps!
